backend/src/main/java/io/debezium/configserver/service/ConnectorIntegratorBase.java

```diff
@@ -22,13 +25,35 @@
 import io.debezium.configserver.model.PropertiesValidationResult;
 import io.debezium.configserver.model.PropertyValidationResult;
 import io.debezium.connector.postgresql.PostgresConnectorConfig;
+import org.jboss.logging.Logger;
 
 public abstract class ConnectorIntegratorBase implements ConnectorIntegrator {
 
+    private static final Logger LOGGER = Logger.getLogger(ConnectorIntegratorBase.class);
+
     protected abstract ConnectorDescriptor getConnectorDescriptor();
 
     protected abstract SourceConnector getConnector();
 
+    public static List<String> enumArrayToList(EnumeratedValue[] input) {
+        List<String> result = new ArrayList<>();
+        for (EnumeratedValue value : input) {
+            result.add(value.getValue());
+        }
+        return result;
+    }
+
+    @Override
+    public Map<String, AdditionalPropertyMetadata> allPropertiesWithAdditionalMetadata() {
+        LOGGER.error(this.getClass().getName() + " must implement ConnectorIntegrator#getAllConnectorProperties!");
+
+        AdditionalPropertyMetadata defaultMetadata = new AdditionalPropertyMetadata(false, ConnectorProperty.Category.CONNECTOR);
+        Map<String, AdditionalPropertyMetadata> result = new HashMap<>();
+        getConnector().config().configKeys().values()
```

Comment:
I'd suggest to use `collect(Collectors.toMap())` instead of forEach.

Reply:
Done, but also refactored in order to have the returned JSON list ordered by the PoC requirement doc